backend/workflow_manager/workflow_v2/workflow_helper.py

```
@@ -206,57 +205,74 @@ def process_input_files(
+                celery_app.signature(
+                    "process_batch_callback",
+                    kwargs={"execution_id": str(workflow_execution.id)},
+                    queue=file_processing_callback_queue,
+                )
             )
             if not result:
                 exception = f"Failed to queue execution task {workflow_execution.id}"
                 logger.error(exception)
                 raise WorkflowExecutionError(exception)
             logger.info(f"Execution {workflow_execution.id} task queued successfully")
         except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)